src/server/hotspot_partition_calculator.h

```diff
@@ -37,6 +51,10 @@ class hotspot_partition_calculator
     void data_aggregate(const std::vector<row_data> &partitions);
     // analyse the saved data to find hotspot partition
     void data_analyse();
+    static void send_hotkey_detect_request(const std::string &app_name,
```

Comment:
Where this function is used? Should it be private? If it's public, why don't make it a free function outside from `hotspot_partition_calculator`?

```
class hotspot_partition_calculator {
};

extern  void send_hotkey_detect_request(...
```

Reply:
> The specific usage will be given after finishing the refactor of hotspot_partition_calculator(#592)
Make it a static function also can be used in other places like shell commands, I don’t know why `static` is changed to `extern` here.